ginga/examples/configs/plugin_SaveImage.cfg

```diff
@@ -0,0 +1,22 @@
+#
+# SaveImage plugin preferences file
+#
+# Place this in file under ~/.ginga with the name "plugin_SaveImage.cfg"
+
+# Default output parameters. Can also be changed in the GUI.
+output_directory = '.'
+output_suffix = 'ginga'
+
+# Clobber existing output files or not
+clobber = False
+
+# Only list modified images from the channel
+modified_only = True
```

Comment:
Does user generally want to save *unmodified* files? Why don't they just do a system copy in that case? I was under the impression that `True` would make more sense, but I can certainly change it to `False` if you think otherwise. Let me know!